docs/DeveloperGuide.adoc

```diff
@@ -109,7 +104,7 @@ Given below is the Sequence Diagram for interactions within the `Logic` componen
 .Interactions Inside the Logic Component for the `delete 1` Command
 image::DeleteSequenceDiagram.png[]
```

Comment:
The diagram shows `AddressBookParser`. Also, since there are multiple directories, the context of `delete` is not explicitly mentioned in the preface. There is also no description to describe the diagram.